Engines/Wine/Plugins/Font smoothing/script.js

```diff
@@ -0,0 +1,35 @@
+include("engines.wine.engine.object");
+
+/**
+ * force the Font smothing
+ * @param {string} mode ("RGB", "BGR" or "Gray Scale")
+ * @returns {Wine} Wine object
+ */
+Wine.prototype.FontSmoothing = function (mode) {
+    var FontSmoothingType;
+    var FontSmoothingOrientation;
+    if (mode == "RGB") {
+        FontSmoothingType = "2";
+        FontSmoothingOrientation = "1";
+    }
+    else if (mode == "BGR") {
+        FontSmoothingType = "2";
+        FontSmoothingOrientation = "0";
+    }
+    else if (mode == "Gray Scale"){
+        FontSmoothingType = "1";
+        FontSmoothingOrientation = "1";
+    }
+
+    var regeditFileContent =
```

Comment:
I think this is the ideal try to introduce template (multiline) strings in our scripts! Such strings make strings like the following much easier readable. For more details see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals